src/output.py

```diff
@@ -106,6 +106,7 @@ def progress(self, futures):
 
         isatty = sys.stdout.isatty()
 
+        sleep_time = 2
```

Comment:
I am not sure I understand the use of this sleep

Reply:
Is this just for polling the response?

Reply:
Yes. Previously it was outputting too much and making the output log even longer.